anti-tracking-threat-model.md

```diff
@@ -0,0 +1,196 @@
+# Anti-tracking Threat Model
+
+WebKit and other browsers are trying to reduce websites' ability to track users
+across the internet. While the success of their effort remains to be seen, we
+don't want Web Packaging to make the problem any more difficult.
+
+This document contains a proposed threat model for the best-case outcome of that anti-tracking effort. We do not yet claim there is consensus about any of:
+
+1. Whether the [attacker capabilities](#attacker-capabilities) are plausible.
+1. Whether it will eventually be possible to frustrate the [attacker goals that
+   we want to frustrate](#attacker-goals-that-we-want-to-frustrate).
+1. Whether the mitigations actually mitigate the attacks.
+1. Whether the costs of the mitigations are worth the benefits.
+1. Probably other things.
+
+Nonetheless, we feel it's important to start with a concrete proposal in mind
+when discussing how to evolve both this threat model and the Web Packaging
+proposals.
+
+Often when we criticize new, technically distinct tracking vectors, we are told
```

Comment:
"We" has become confused here. It shouldn't be the same "we" criticizing tracking vectors and "we" are told.